injected/unit-test/verify-artifacts.js

```diff
@@ -42,7 +34,7 @@ const checks = {
     },
     integration: {
         file: join(BUILD, 'integration/contentScope.js'),
-        tests: [{ kind: 'containsString', text: 'trackerLookup: {', includes: true }],
+        tests: [{ kind: 'containsString', text: 'trackerLookup: ', includes: true }],
```

Comment:
This isn't really validating anything anymore. I can't remember why we added it tbh?

Perhaps we should just remove the test or change to:
```suggestion
        tests: [{ kind: 'containsString', text: 'define_import_meta_trackerLookup_default = {', includes: true }],
```

Reply:
I'll remove it